Documentation/contributors/design-proposals/operator-validation-library.md

```diff
@@ -53,3 +53,191 @@ There is also a Command Line Tool that serves as an example of how a user can in
 
 - How do we deal with different versions of the CSV type?
   - Ideas: include a version of the library alongside the CSV type definition so that it is versioned in the same way the CSV is versioned.
+
+# Fake README
+
+## Installing
+
+### Command Line Tool
+
+You can interact with this library with a command line tool. 
+
+You can install the `operator-verify` tool from source using:
+
+```
+$ go install
+```
+
+```bash
+$ echo $PATH
+```
+
+If you do not have your workspace's `bin` subdirectory in your `$PATH`, 
+
+```bash
+$ export PATH=$PATH:$(go env GOPATH)/bin
+```
+
+This adds your workspace's bin subdirectory to your PATH. As a result, you can use the `operator-verify` tool anywhere on your system. Otherwise, you would have to `cd` to your workspace's `bin` directory to run the executable. Verify that we can find `operator-verify` in our new `PATH`:
+
+```bash
+$ which operator-verify
+```
+
+This should return something like:
+
+```
+~/go/bin/operator-verify
+```
+
+To verify that the library installed correctly, use it to validate your ClusterServiceVersion yaml,
+
+```
+$ operator-verify csv /path/to/filename.yaml
+```
+
+## Usage
+
+The command line tool's `help` command gives the following output:
+
+```text
+operator-verify is a command line tool for the Operator Manifest Verification Library. This library provides functions to validate the operator manifest bundles against Operator-Lifecycle-Manager's clusterServiceVersion type, customResourceDefinitions, and package yamls. Currently, this application supports static validation of both individual operator files and the manifest bundle. 
+
+Usage:
+  operator-verify [flags]
+  operator-verify [command]
+
+Available Commands:
+  help        Help about any command
+  csv         Validate CSV against OLM's type
+  crd         Validate manifest CRDs
+  package     Validate package yaml
+  manifest    Validate operator manifest bundle  
```

Comment:
what about having it point to an image too? like you could use a flag to specify whether the manifest is local or remote. Thoughts?

Reply:
You mean it takes an image and extracts the bundle out of it?

Reply:
yes